src/api/messages/post.js

```diff
@@ -0,0 +1,12 @@
+import { collections } from '../../firebase/index';
+
+export default function (caseId, sender, content) {
+  return collections.cases
+    .doc(caseId)
+    .collection("messages")
+    .add({
+      created: new Date(),
```

Comment:
Handling the created field with values could be troublesome to say the least.

From reading the Firestore documentation I think these two alternatives could be a little bit more reliable:

- [Data types](https://firebase.google.com/docs/firestore/manage-data/add-data#data_types)
- [Server timestamp](https://firebase.google.com/docs/firestore/manage-data/add-data#server_timestamp)

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/4506c0292a3a878f3d4b9691d66c8c57f5c4c3df 👍🏼